### PR TITLE
Packs page

### PR DESCRIPTION
**Pull request by taqh**
- Built the packs page 
- Uses dummy data
- Client side filtering with URL params
Packs page
With filter applied
Pack details
Light mode 1
Light mode 2

### CODE REVIEW

**Comment by mezotv** [comment]
Is there a chance you can get rid of the other commits not related to this page? Also can we use the figmas colors spacing etc especially on the cards and tags?
image

File: app/packs/_components/QuestionPack.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        </div>
+      </CardContent>
+      <CardFooter className="grid grid-cols-2 gap-4 lg:gap-10">
+        <Button
+          onClick={() => likePack(pack.id)}
+          variant="secondary"
+          className="w-full"
+        >
+          <Heart className="mr-2 h-4 w-4 text-brand-customGrayText fill-brand-customGrayText" />
+          <span>{pack.likes} Likes</span>
```
Comment (fb-sean) [inline app/packs/_components/QuestionPack.tsx]: Would suggest to do .toLocaleString() at likes

**Comment by fb-sean** [comment]
I think it looks better with just the search bar like without "Question" 
image

File: app/packs/_components/QuestionPack.tsx (hunk 1 above)
Comment (taqh) [reply app/packs/_components/QuestionPack.tsx]: will do

**Comment by taqh** [comment]
## Made changes using the figma colors and spacing as requested

### Light 
Screenshot 2024-10-03 151937
Screenshot 2024-10-03 152003

### Dark
Screenshot 2024-10-03 152020
Screenshot 2024-10-03 152035

**Review by dpaulos6 — APPROVED** [review]

**Pull request merged**